.github/workflows/buildAndDeployPhar.yml

```diff
@@ -0,0 +1,52 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: Build and deploy PHAR
+on:
+  push:
+    tags:
+    - "*"
+jobs:
+  build:
+    name: Build PHAR-File
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Build Phar
+      uses: MilesChou/composer-action/7.2/install@master
+      with:
+        args: buildphar
+
+    - name: Sign Phar
+      env:
+        DECRYPT_KEY: ${{ secrets.DECRYPT_KEY }}
+        SIGN_KEY: ${{ secrets.SIGN_KEY }}
+      run: |
+        export GPG_TTY=$(tty)
+        echo $DECRYPT_KEY
+        echo Decrypting key
+        gpg --batch --yes --passphrase $DECRYPT_KEY KeepAChangelogSigning.asc.gpg && gpg --batch --yes --import KeepAChangelogSigning.asc
+        echo Signing Artifact
+        gpg --batch --yes --pinentry-mode loopback --passphrase $SIGN_KEY -u 5A30219F03D3A2A5 --armor --detach-sig build/keep-a-changelog.phar
+
+    - name: Prepare Release-Info
+      run: |
+        echo ::set-env name=RELEASE_VERSION::${GITHUB_REF/refs\/tags\//}
+        ./build/keep-a-changelog.phar version:show ${{ env.RELEASE_VERSION }} > release.note
+
+    - name: Create release
+      uses: ncipollo/release-action@v1
+      with:
+        artifacts: "build/*"
+        bodyfile: release.note
+        name: "keep-a-changelog ${{ env.RELEASE_VERSION }}"
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+#    - name: Create GitHub release
+#      uses: Roang-zero1/github-upload-release-artifacts-action@master
+#      with:
+#        args: "build/"
+#      env:
+#        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I assume that ncipollo/release-action uploads the phar when creating the release?

If so, can we remove this section before I merge?

Reply:
It does the complete creation of the release including adding the artifacts. So commenting out the one line referencing `build` should be enough to not upload the artifacts. Or shall I comment out the whole step to automatically create the release from the tag?